portal/portal-render-engine-impl/impl/src/bundle/trinity/options.config

```diff
@@ -0,0 +1,11 @@
+include-bottom: true
```

Comment:
Will we need to add to this if we add new velocity templates for trinity?

Reply:
As I understand it this does not link directly to the templates. Rather it controls if related data of the portal's sections is added to the render context of the portal. As the render context I understand the the data that is available to the template when the page renders. I would be happy to learn more about it though. But as the render context changes, this might (and probably will) also affect some templates are loaded. [Here](https://github.com/sakaiproject/sakai/blob/53f53ee631ce7f3e55b1f844e56546a4c207f5e6/portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java#L1593) you can see what the line that you commented on would affect. As I see it, if it makes sense to include/excluse a section a template by preference it should be reflected in that file. I guess it also makes sense to rethink, what we should to have in this options as well, as if someone actually uses this option, It shouldn't break other functionality, as i suspect, it can.